sklearn/tree/_criterion.pxd

```diff
@@ -31,13 +31,13 @@ cdef class Criterion:
     cdef SIZE_t n_missing                # Number of missing values for the feature being evaluated
     cdef bint missing_go_to_left         # Whether missing values go to the left node
 
-    cdef SIZE_t n_outputs                # Number of outputs
-    cdef SIZE_t n_samples                # Number of samples
-    cdef SIZE_t n_node_samples           # Number of samples in the node (end-start)
-    cdef double weighted_n_samples       # Weighted number of samples (in total)
-    cdef double weighted_n_node_samples  # Weighted number of samples in the node
-    cdef double weighted_n_left          # Weighted number of samples in the left node
-    cdef double weighted_n_right         # Weighted number of samples in the right node
+    cdef SIZE_t n_outputs                 # Number of outputs
+    cdef SIZE_t n_samples                 # Number of samples
+    cdef SIZE_t n_node_samples            # Number of samples in the node (end-start)
+    cdef double weighted_n_samples        # Weighted number of samples (in total)
+    cdef double weighted_n_node_samples   # Weighted number of samples in the node
+    cdef double weighted_n_left           # Weighted number of samples in the left node
+    cdef double weighted_n_right          # Weighted number of samples in the right node
     cdef double weighted_n_missing       # Weighted number of samples that are missing
```

Comment:
```suggestion
    cdef double weighted_n_missing        # Weighted number of samples that are missing
```
Not sure I understand why the extra space got added on the previous lines, but if we add it there we should do so here as well. For consistency :D